src/pages/ticketConfirmation.tsx

```diff
@@ -294,7 +296,11 @@ export const buildTicketConfirmationElements = (
                     },
                     {
                         name: `Product - ${product.productName}`,
-                        content: `Validity - ${startCase(product.productValidity)}`,
+                        content: removeExcessWhiteSpace(
```

Comment:
Do we need removeExcessWhiteSpace on here? Just thinking that we control the productValidity and serviceEndTime, so we know there will be no excess whitespace.

Reply:
have done this and added undefined instead of '' in the ternary

Reply:
Oh, whats the reason for that? I dont think we need to add undefined.